tests/integration/server.py

```diff
@@ -24,6 +24,19 @@
 
 from flask import Flask, request, jsonify
 
+# set up logging module to write to Stackdriver
+client = google.cloud.logging.Client()
```

Comment:
I don't think this needs to be a global variable, and I dislike having this much code at the top level.  I think you can move these four lines under `if __name__ == '__main__'`